docs/maven/build-integration.md

```diff
@@ -0,0 +1,192 @@
+  [ ![jSparrow license](/img/maven/jenkins-jsparrow-license.png) ](/img/maven/jenkins-jsparrow-license.png)
+    * Kind: Select "Secret text"
+    * Scope: This needs to be "Global", otherwise the pipeline will not have access
+    * Secret: This is where the jSparrow license key has to be provided.  
+    Note: The content of this variable cannot be viewed anymore after putting it here and it will not appear in logs or echo statements.
+    * ID: This is the key with which the secret text can be referenced in the pipeline
+    * Description: It is always a good practice to provide a description
+
+### jSparrow Config File
+
+Customization about which rules to apply or which resources to skip are done in a config file (`jsparrow.yml`), as described in detail in [Plugin Configuration](/maven/getting-started.html#plugin-configuration).
+
+By default, this configuration file is expected to reside in the project's root directory. Alternatively, there is the option to fully manage the jSparrow configuration in Jenkins. This might have the following advantages:
+* The `jsparrow.yml` does not need to be committed to the project's SCM repository
+* Jenkins administrators can manage the configuration globally
+
+The config file can be added as follows:
+
+1. Navigate to *Manage Jenkins* -> *Managed files* and click "Add a new Config"
+1. Add a new "Custom file" and provide a meaningful ID
+1. This is a description of the fields of the following screen:  
+  [ ![jsparrow.yml](/img/maven/jenkins-jsparrow-yml-content.png) ](/img/maven/jenkins-jsparrow-yml-content.png)
+    * ID: The key with which the file will be referenced within the pipeline
+    * Name: The file name
+    * Comment: An optional description
+    * Content: The configuration according to the [Plugin Configuration](/maven/getting-started.html#plugin-configuration)
+
+## Pipeline Configuration
+
+With [Multibranch Pipelines](https://www.jenkins.io/doc/book/pipeline/multibranch/), Jenkins offers a way to run builds on all branches of a repository. This is very useful when following a branching model, such as [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow), where different types of branches have different purposes. The current branch can be determined during runtime (`env.BRANCH_NAME` or via [Built-in Condition](https://www.jenkins.io/doc/book/pipeline/syntax/#built-in-conditions)), making it possible to process certain branches differently. 
+
+In this example, a declarative Multibranch Pipeline will be configured to run the jSparrow Maven plugin on feature branches. In Gitflow, a feature branch should only be merged into develop (and master) after being reviewed in a pull request. 
+
+### Step-by-step Description
+
+Here, the necessary pipeline file will be described one part at a time. Click on [Jenkinsfile.groovy](https://gist.github.com/luigiwerzowa/aafba402bd36faa573f48a26fc056289) for the full file. 
+
+#### Tools Configuration
```

Comment:
IMO it can be avoided to go so deep in subsections.

Reply:
I will remove the "Spet-by-step Description" headline.